finagle-core/src/main/scala/com/twitter/finagle/dispatch/ServerDispatcher.scala

```diff
@@ -103,8 +114,30 @@ class SerialServerDispatcher[Req, Rep](
     service.close()
   }
 
-  protected def dispatch(req: Req, eos: Promise[Unit]) =
+  override def dispatch(req: Req, eos: Promise[Unit]) =
     service(req) ensure eos.setDone()
 
-  protected def handle(rep: Rep) = trans.write(rep)
+  override def handle(rep: Rep) = trans.write(rep)
+}
+
+/**
+  * A wrapper type around a 
+  * [[com.twitter.finagle.dispatcher.ServerDispather ServerDispatcher]]
```

Comment:
let's also be clear about what kinds of ServerDispatchers it can wrap.  this is only OK for RPC, right?